src/controllers/404.ts

```diff
@@ -0,0 +1,13 @@
+export async function load404Page() {
+  const contentArea = document.getElementById("content-area");
+
+  if (contentArea) {
+    try {
+      const response = await fetch("./404.html");
```

Comment:
Why wouldn't you prefetch at build time aka import a string this doesn't make sense to me